test/DocumentFormat.OpenXml.Tests/Common/ITestOutputHelperExtenstions.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using Xunit.Abstractions;
+
+namespace DocumentFormat.OpenXml.Tests.Common.OutputHelperExtensions
+{
+    public static class ITestOutputHelperExtenstion
+    {
+        static public void WriteObjectToTempFile(this ITestOutputHelper output, object obj)
```

Comment:
You can mark this with the attribute `[Conditional("DEBUG")]` and then you don't need to if/def - it'll only happen on debug